app/src/gui/components/notebook/index.tsx

```diff
@@ -233,100 +276,120 @@ export default function NotebookComponent(props: NotebookComponentProps) {
-                      </TableRow>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>
-                            Lead Institution
-                          </Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'lead_institution'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>
-                            Project Lead
-                          </Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'project_lead'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                      <TableRow>
-                        <TableCell>
-                          <Typography variant={'overline'}>
-                            Last Updated
-                          </Typography>
-                        </TableCell>
-                        <TableCell>
-                          <MetadataRenderer
-                            project_id={project.project_id}
-                            metadata_key={'last_updated'}
-                            chips={false}
-                          />
-                        </TableCell>
-                      </TableRow>
-                    </TableBody>
-                  </Table>
-                </TableContainer>
-              </Grid>
-              <Grid item xs={12} sm={12} md={12} lg={4}>
-                <RangeHeader
-                  project={project}
-                  handleAIEdit={handleNotebookTabChange}
+                  Edit
+                </Typography>
+              </IconButton>
+            </Box>
+
+            <Box sx={{p: 2}}>
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Name:</strong>{' '}
+                <MetadataRenderer
+                  project_id={project.project_id}
+                  metadata_key={'name'}
+                  chips={false}
                 />
-              </Grid>
-            </Grid>
+              </Typography>
+              <Typography
+                variant="body1"
+                gutterBottom
+                sx={{marginBottom: '16px'}}
+              >
+                <strong>Description:</strong>{' '}
+                <MetadataRenderer
```

Comment:
As it happens, we made this metadata field a Markdown field but have not got around to doing the markdown formatting here yet.  You could maybe add that in here. We already import the `markdown-it` module to format markdown in the RichText form field, so would just need to apply that here.

Reply:
Have made changes to use markdownit to display the fields in that format.